manifests/install.pp

```diff
@@ -18,7 +18,7 @@
   $python = $::python::version ? {
     'system' => 'python',
     'pypy'   => 'pypy',
-    default  => "${python::version}",
```

Comment:
Either way, that variable will still be an int or a str based on it's contents.  Quoting a variable will not cast it to a string.

```
root@testbox:~# cat test.pp 
$version=123
warning(type($version))
warning(type("${version}"))
```

```
root@testbox:~# puppet apply test.pp 
Warning: Scope(Class[main]): integer
Warning: Scope(Class[main]): integer
```